sdk/python/tests/compiler/testdata/condition.py

```diff
@@ -13,29 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import kfp
-from kfp import dsl
 
-def print_op(msg):
-    """Print a message."""
-    return dsl.ContainerOp(
-        name='Print',
-        image='alpine:3.6',
-        command=['echo', msg],
-    )
+import kfp.dsl as dsl
+
+
+class FlipCoinOp(dsl.ContainerOp):
+
+  def __init__(self, name):
+    super(FlipCoinOp, self).__init__(
+      name=name,
+      image='python:alpine3.6',
+      command=['sh', '-c'],
+      arguments=['python -c "import random; result = \'heads\' if random.randint(0,1) == 0 '
+                 'else \'tails\'; print(result)" | tee /tmp/output'],
+      file_outputs={'output': '/tmp/output'})
+
+
+class PrintOp(dsl.ContainerOp):
+  
+  def __init__(self, name, msg):
+    super(PrintOp, self).__init__(
+      name=name,
+      image='alpine:3.6',
+      command=['echo', msg])
     
 
 @dsl.pipeline(
-    name='Conditional Example Pipeline',
-    description='Shows how to use dsl.Condition().'
+  name='pipeline flip coin',
+  description='shows how to use dsl.Condition.'
 )
-def conditional_pipeline(num: int = 5):
+def flipcoin():
```

Comment:
We need to add this into our list of unit test at https://github.com/kubeflow/kfp-tekton/blob/master/sdk/python/tests/compiler/compiler_tests.py